.github/workflows/tests.yml

```diff
@@ -0,0 +1,52 @@
+# This workflow will install Python dependencies, run tests with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: commcare-cloud tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+
+    env:
+      ANSIBLE_ROLES_PATH: "~/.ansible/roles"
+      CI_TEST: 1
+      TEST: main
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v3
+      with:
+        python-version: "3.6"
+
+    - name: Install dependencies
+      run: |
+        sudo apt-get clean
+        sudo apt-get update -qq
+        source control/init.sh
+        # Note: the installs must be done in the following order, otherwise the installation of commcare-cloud
+        # tries to install from the commcare-cloud-bootstrap.egg_info directory
+        pip install -e .[test]
+
+        # Python for deploy-stack in .tests/tests.sh
+        # May be removed when this file is updated to match global_vars/all.yml:python_version
+        curl -sSf --retry 5 -o python-3.9.tar.bz2 'https://storage.googleapis.com/travis-ci-language-archives/python/binaries/ubuntu/18.04/x86_64/python-3.9.tar.bz2'
+        sudo tar xjf python-3.9.tar.bz2 --directory /
```

Comment:
We're currently targeting python 3.6, why does this part require 3.9?
If that's necessary, I would prefer that we install it via apt by e.g. using the deadsnakes repository like we currently do for the deployment of HQ, instead of relying on an obscure Travis binary.

Reply:
The way these tests are currently run, we run our cchq `deploy-stack` command, limited to a small set that includes `localsettings.py` changes, _on the local VM_ using the ansible `local` connection in the test environment inventory. Even though `commcare-cloud` currently runs on 3.6, it requires python 3.9 to be running on the "remote" machine. In this case, the remote machine is itself, so it needs both 3.6 and 3.9.

This is copied straight from the `.travis.yml` this file is replacing.

Reply:
> If that's necessary, I would prefer that we install it via apt by e.g. using the deadsnakes repository like we currently do for the deployment of HQ, instead of relying on an obscure Travis binary.

This is a good suggestion. Eventually I would like to see us put the target environment in a container within this container, if that's practical, to isolate the environment. Then we could just use our ansible scripts themselves to do it without worrying about clobbering other things in the runner VM itself.

Reply:
As the one who added this to `.travis.yml`, I agree, it would be nice to install this in a more standard way.

Reply:
I'm going to punt this to a followup and keep this PR focused on switching CI systems, but yeah I agree there's a clear better way to do this